packages/web/src/pages/settings-page/components/mobile/SlidingPages.tsx

```diff
@@ -0,0 +1,83 @@
+import { Children, ReactNode, useState } from 'react'
+
+import { Flex } from '@audius/harmony'
+import { ResizeObserver } from '@juggle/resize-observer'
+import { animated, useTransition } from '@react-spring/web'
+import useMeasure from 'react-use-measure'
+
+const defaultTransitions = {
+  initial: { opacity: 1, transform: 'translate3d(0%, 0, 0)' },
+  enter: { opacity: 1, transform: 'translate3d(0%, 0 ,0)' }
+}
+const getSwipeTransitions = (direction: 'back' | 'forward') =>
```

Comment:
nice! like you mentioned, would be great to build this as a "navigation" type component like native

Reply:
💯 just give me some more caffeine 😂